tests/test_suites/ShortSystemTests/test_global_io_limiting_read_write.sh

```diff
@@ -16,7 +19,11 @@ CHUNKSERVERS=3 \
 	MOUNT_EXTRA_CONFIG="sfscachemode=NEVER" \
 	setup_local_empty_saunafs info
 
-truncate -s1M "${info[mount0]}/read"
+if is_windows_system; then
+	FILE_SIZE=1048576 file-generate "${info[mount0]}/read"
+else
+	truncate -s1M "${info[mount0]}/read"
+fi
```

Comment:
Please comment about why is different (truncate).

Reply:
Looks like old client version did not support it. Probably at some point I thought it was WSL1 fault of not supporting truncate (like chmod and chown) in a drive mounted using drvfs.

Current client version passed the test with the truncate command. Will revert it.